search-app/src/cmr/search/services/query_execution/facets/facets_results_feature.clj

```diff
@@ -22,7 +22,7 @@
   "Used to build an aggregation to get a count of unique concepts included in the current nested
   aggregation."
   {:reverse_nested {}
-   :aggs {:concept-id {:terms {:field :concept-id :size 1}}}})
+   :aggs {:concept-id {:cardinality {:field :concept-id}}}})
```

Comment:
Solution change number 2 - switch from terms aggregation to cardinality for cycle and pass nested aggregations.